### DISCUSSION TITLE
How can I change the language?

### DISCUSSION THREAD

**Discussion by ls1146** [liedekef/events-made-easy#48] [Q&A] [answered]
I don't quite understand right now how I can change Events Made Easy back to German. Since the security update I use it in English, but my colleagues would be very happy if I could change it back to German. Could you tell me how?
Thank you so much!

**Comment by liedekef** [comment]
... just set your site language to German? Or change the lang of the admin interface to German? EME provides the languages in the langs subdir and events-made-easy-de_DE.po is present there.

**Reply by ls1146** [reply]
the site language is in german ... its only the admin interface of EME thats in English - where do i change the admin interface lang?

**Comment by liedekef** [comment]
Check your WP profile.

**Reply by ls1146** [reply]
There is everything in German.  I thought that might be a setting form EME  because everything else is in German in our Dashboard

**Comment by liedekef** [comment]
And what is the locale then? Because I tested it with English, Dutch, French and German (Deutsch) and it all works as expected.

**Reply by ls1146** [reply]
if i change it to dutch nothing happens as well..
![2023-01-31 15_57_19-Einstellungen › Allgemein ‹ KONSI — WordPress](https://user-images.githubusercontent.com/122777891/215795684-55693812-8347-4dc3-97eb-56830f6751b7.png)

**Comment by liedekef** [comment]
It immediately works here, on a default site, when I change the language. Is the subdir langs present in the plugin folder? Are the language files present in there? Which EME version are you using? Are other plugins conflicting?
Also, I support nl_NL and nl_BE , but I don't know what language code the "Formeel" part corresponds with for Dutch.

**Reply by ls1146** [reply]
It looks like this - so I think the languge "file" is not in there? Do I have to upload it by myself?
I downloaded a version here at Github, uploaded this in wordpress and then it was in English and the changes in settings doesn't do anything. Since I uploaded the plugin once after the security problem, i make the updates in Wordpress directly. Now it is version 2.3.47

![grafik](https://user-images.githubusercontent.com/122777891/216037430-b86e0a9d-9188-4415-955c-3f1953d1b392.png)

**Reply by ls1146** [reply]
i tried to upload it again, i geht the message that the upload was successful but there are still no language files.
But i can see alle the files after download the latest release in the zip ordner..

**Reply by liedekef** [reply]
maybe a permission issue on your folder?

**Reply by ls1146** [reply]
maybe i know now why the problem occurs : i don't know why but i have 2 versions of EME - i just made a backup, then i tried to delete one. That does not work while the other is activated (i waited at least 15mins). If if deactivate both versions and delete one, all my data is gone (i it like i installed the first time - and it doesn't matter whichs version i delete). Do you know how i may delete one and keep all my data?

![grafik](https://user-images.githubusercontent.com/122777891/216074139-bd9431b8-bc0c-401f-9fda-6d3a93140800.png)

**Comment by liedekef** [comment, marked as answer]
You installed by copying over the existing folder to a backup and then installed the new via manual unzip (my guess). This is not just an EME issue, any plugin will result in this result then. For you: just removing the files of the old plugin should be sufficient.

**Reply by ls1146** [reply]
what do you mean by "files of the old plugin"?

**Comment by liedekef** [comment]
go in the plugins folder and delete the directory containing the files of EME from the old version.

**Reply by ls1146** [reply]
so now i deleted the old version, but it's still not getting translated (but all my data is there :)). i really don't know what else i can do ..

**Reply by ls1146** [reply]
so i don't know why, but suddenly it works! Thank you for your great work and help!
